bucket/wechat.json

```diff
@@ -36,9 +35,9 @@
     },
     "checkver": {
         "url": "https://pc.qq.com/detail/8/detail_11488.html",
-        "regex": "WeChat([\\d.]+)\\.exe"
+        "regex": "WeChat(\D*)([\d.]+)\.exe"
     },
     "autoupdate": {
-        "url": "https://webcdn.m.qq.com/spcmgr/download/WeChat$version.exe#/dl.7z"
+        "url": "https://dldir1.qq.com/weixin/Windows/WeChatSetup.exe#/dl.7z"
```

Comment:
the url breaks the hash checks while upstream publish a new version without updating `checkver.url` page.